sdk/tracing/src/test/java/io/opentelemetry/sdk/trace/SpanBuilderSdkTest.java

```diff
@@ -139,7 +139,7 @@ void truncateLinkAttributes() {
     RecordEventsReadableSpan span = (RecordEventsReadableSpan) spanBuilder.startSpan();
     try {
       assertThat(span.toSpanData().getLinks())
-          .containsExactly(
+          .containsExactlyInAnyOrder(
```

Comment:
Not a big deal, but surprised by the change since looks like only one link :)

Reply:
isn't needed. this snuck in while I was hacking on stuff, and forgot to revert.

Reply:
fixed, and thanks for the sharp eye!